src/examples/example.tsx

```diff
@@ -68,6 +65,20 @@ export class Example {
         ];
     }
 
+    private renderExample() {
+        const ExampleComponent = this.name;
+        return this.codeOnly
+            ? null
+            : [
+                  <limel-config
+                      config={{ iconPath: `${BASE_URL}public/stencil/` }}
+                  />,
+                  <div class="example">
+                      <ExampleComponent />
+                  </div>,
+              ];
```

Comment:
It's a bit hard to read this code and I think it's because of the ternary operator. I think we should avoid using the it unless the expression is on a single line. 

```suggestion
    if (this.codeOnly) {
        return;
    }

    const ExampleComponent = this.name;    
    return [
        <limel-config
            config={{ iconPath: `${BASE_URL}public/stencil/` }}
        />,
        <div class="example">
            <ExampleComponent />
        </div>
    ];
```

Reply:
Yeah, that makes sense 👍

Reply:
:zap:

Interesting… I think the conversation was auto-resolved because I committed the suggestion…? At least I don't think I resolved it manually…